src/cmd/cli/command/compose.go

```diff
@@ -105,12 +106,16 @@ func makeComposeUpCmd() *cobra.Command {
 			const targetState = defangv1.ServiceState_DEPLOYMENT_COMPLETED
 
 			go func() {
-				services := make([]string, len(deploy.Services))
-				for i, serviceInfo := range deploy.Services {
-					services[i] = serviceInfo.Service.Name
+				monitoredServices := make([]string, 0, len(deploy.Services)-len(managedResources))
+				for _, serviceInfo := range deploy.Services {
+
+					// do not monitor managed resources, they do not currently update their status
+					if !slices.Contains(managedResources, serviceInfo.Service.Name) {
+						monitoredServices = append(monitoredServices, serviceInfo.Service.Name)
+					}
 				}
 
-				if err := cli.WaitServiceState(tailCtx, provider, targetState, deploy.Etag, services); err != nil {
+				if err := cli.WaitServiceState(tailCtx, provider, targetState, deploy.Etag, monitoredServices); err != nil {
```

Comment:
I think we should probably handle this in the backend: when we get `Subscribe` for a managed service, we return a status that's considered complete

Reply:
Wait, this is for BYOC, but we can still handle it inside `Subscribe`.